CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ These are the section headers that we use:
 
 ### Added
 
+- Added `get_metrics` function in the SDK to get `FeedbackDataset` metrics calling `GET /api/v1/me/{dataset_id}/metrics` endpoint ([#3469](https://github.com/argilla-io/argilla/pull/3469))
```

Comment:
I think we can remove this line since is too techie detail for a changelog 
```suggestion
```

By the way, we can add other in the changed and deprecated sections, referring the new `add_records` behaviour. or the push_to_argilla and other deprecated methods